README.md

```diff
@@ -5,7 +5,7 @@
 [![Build Status][travis-image]][travis-url]
 [![Test Coverage][coveralls-image]][coveralls-url]
 
-Merge objects using descriptors.
+Merge source object properties onto a destination object, and preserve descriptors.  Descriptors are configuration data associated to each property such as whether the property is `writable`, `enumerable`, and/or `configurable`.  This data will be preserved in the destination object.
```

Comment:
Please hard wrap the lint to approx 80 chars. And use a single space instead of double space.

Reply:
So maybe it should explain which of the source properties are going to be copied into the destination as well.

Reply:
Good point.  That would seem to be the object's "own" properties.